browser/todo-list.md

```diff
@@ -3,291 +3,248 @@
+
+This app highlights the following platform functionality:
+
+- Generate Secret Key with associated Blockstack username to authenticate app
+- Add, edit and delete encrypted app data with Gaia
+- Decrypt data on Gaia for public sharing by URL
+- Unauthenticate and re-authenticate app with Secret Key
+
+[Preview the app](https://todos.blockstack.org) or [view its code on GitHub](https://github.com/blockstack/blockstack-todos).
+
+Existing familiarity with React is recommended for reviewing this app's code.
 
 * TOC
 {:toc}
 
-{% include note.html content="On macOS, Blockstack requires macOS High Sierra. This tutorial was written on macOS High Sierra 10.13.4. If you use a Windows or Linux system, you can still follow along. However, you will need to \"translate\" appropriately for your operating system. Additionally, this tutorial assumes you are accessing the Blockstack Browser web application via Chrome. The application you build will also work with a local installation and/or with browsers other than Chrome. " %}
+### Install and run the app
 
-## Before you begin
+You must have recent versions of Git and [Node.js](https://nodejs.org/en/download/) (v12.10.0 or greater) installed already.
 
-The application you build is a React application that is completely decentralized and server-less. While not strictly required to follow along, basic familiarity with React is helpful. When complete, the app is capable of the following:
 
-* authenticating users using Blockstack
-* posting new statuses
-* displaying statuses in the user profile
-* looking up the profiles and statuses of other users
+1. Install the code and its dependencies:
 
-For this tutorial, you will use the following tools:
+    ```
+    git clone https://github.com/blockstack/blockstack-todos
+    cd blockstack-todos
+    npm install
+    ```
 
-* your workstation's command line
-- Node.js  v10 or higher is recommended the minimum supported version is Node.js v8.
+2. Run the application:
 
-The basic identity and storage services are provided by blockstack.js. To test the application, you need to have already registered a Blockstack ID.
+    ```bash
+    $ npm run start
+    ```
+    
+    You should see output similar to the following:
 
-### Verify you have Node.js and its tools installed
+    ```bash
+      Compiled successfully!
 
-The tutorial relies on Node.js and its `npx` or `npm` tools. Before you begin, verify you have the correct version of Node.js and its tools installed.
+      You can now view bs-todo in the browser.
 
-```bash
-$ node -v
-v12.10.0
-$ which npm npx
-/usr/local/bin/npm
-/usr/local/bin/npx
-```
+        http://localhost:3000/
 
-If you don't, make sure they are installed.
+      Note that the development build is not optimized.
+      To create a production build, use npm run build.
+      ```
 
-### Make sure you have a Blockstack ID
+2. Open your local browser to [`http://localhost:3000`](http://localhost:3000) if it doesn't open automatically.
```

Comment:
I'd say something similar to "your browser will automatically open up the To Do app, available at http://localhost:3000"